### ISSUE TITLE
Error when loading pretrained vgg and alexnet 

### ISSUE THREAD

**Issue by DmitryUlyanov** [DmitryUlyanov/deep-image-prior#10]
When loading pretrained vgg and alexnet in python 3.6: 
```
'ascii' codec can't decode byte 0xc3 in position 875: ordinal not in range(128)
```

**Comment by babaozhouy5** [comment]
How did you load pretrained vgg or alexnet?

I also try to load vgg16 for preceptual loss :). And it works well. Here is my method:

    # python3.6
    import torchvision.models as models
    vgg = models.vgg16(pretrained=True).features # except the classification layers

You may look [here](https://github.com/pytorch/vision/blob/master/torchvision/models/vgg.py) or [here](https://github.com/pytorch/examples/blob/master/fast_neural_style/neural_style/vgg.py) for some ideas.

**Comment by DmitryUlyanov** [comment]
I've converted VGG-19 and Alexnet from Caffe to Torch7 and then to Pytorch. Did so because other papers use these nets pretrained with Caffe.

Models from pytorch's torchvision are different to the reference models, yet it is ok to use them if your aim is not to compare to other works.

**Comment by tumble-weed** [comment]
Hi,
have you made the code snippet converting caffe to pytorch available somewhere? 
thanks
